test/lib/ansible_test/_data/sanity/validate-modules/validate_modules/main.py

```diff
@@ -32,10 +32,17 @@
 
 from collections import OrderedDict
 from contextlib import contextmanager
-from distutils.version import StrictVersion
+from distutils.version import StrictVersion, LooseVersion
 from fnmatch import fnmatch
+
 import yaml
 
+try:
+    import semantic_version
+    HAS_SEMANTIC_VERSION = True
+except Exception as e:
+    HAS_SEMANTIC_VERSION = False
```

Comment:
Since `semantic_version` is listed in the requirements I don't think we should be checking for it here. Just let it fail as it would for any other missing requirement.

Not special casing this will also prevent inconsistency in the reported errors. The errors reported should be consistent between environments, which means tests either need to fail (traceback) due to missing dependencies, or be skipped entirely.